nativelib/src/main/scala/java/lang/Object.scala

```diff
@@ -14,22 +14,22 @@ class _Object {
     getClass.getName + "@" + Integer.toHexString(hashCode)
 
   def _getClass(): _Class[_] =
-    new _Class(Type.get(this))
+    new _Class(runtime.getType(this))
```

Comment:
Not really related to this PR, but this is probably not correct wrt. the reference equality of `java.lang.Class`es. You should have a global map of already created instances of `Class`.